### PR TITLE
[WWB]: Add support for text-to-image GenAI pipelines

### PR DESCRIPTION
**Pull request by AlexKoff88**

### CODE REVIEW

**Comment by AlexKoff88** [comment]
Test is failed due to the known issue "packages/openvino_genai/libopenvino_tokenizers.so: cannot open shared object file: No such file or directory"

**Comment by ilya-lavrenov** [comment]
> Test is failed due to the known issue "packages/openvino_genai/libopenvino_tokenizers.so: cannot open shared object file: No such file or directory"

Do we have a ticket for it? I have not seen such issue

File: tools/who_what_benchmark/whowhatbench/text2image_evaluator.py [hunk 1]
```diff
@@ -26,6 +27,17 @@
...
 
+class Generator(openvino_genai.Generator):
+    def __init__(self, seed, rng, mu=0.0, sigma=1.0):
+        openvino_genai.Generator.__init__(self)
+        self.mu = mu
+        self.sigma = sigma
+        self.rng = rng
+
+    def next(self):
+        return torch.normal(torch.tensor(self.mu), self.sigma, generator=self.rng)
```
Comment (eaidova) [inline tools/who_what_benchmark/whowhatbench/text2image_evaluator.py:38]: it is expected that generator returns torch tensor on genai side?

File: tools/who_what_benchmark/whowhatbench/text2image_evaluator.py [hunk 2]
```diff
@@ -144,13 +166,16 @@ def default_gen_image_fn(model, prompt, num_inference_steps, generator=None):
 
         if not os.path.exists(image_dir):
             os.makedirs(image_dir)
+
+        print(gen_image_fn)
```
Comment (eaidova) [inline tools/who_what_benchmark/whowhatbench/text2image_evaluator.py:170]: looks like debug print

File: tools/who_what_benchmark/whowhatbench/text2image_evaluator.py (hunk 1 above)
Comment (ilya-lavrenov) [reply tools/who_what_benchmark/whowhatbench/text2image_evaluator.py:38]: it should return float number

File: tools/who_what_benchmark/whowhatbench/wwb.py [hunk 3]
```diff
@@ -278,6 +302,18 @@ def parse_args():
         action="store_true",
         help="Use LLMPipeline from transformers library to instantiate the model.",
     )
+    parser.add_argument(
+        "--image-size",
```
Comment (eaidova) [inline tools/who_what_benchmark/whowhatbench/wwb.py:306]: what do you think about addition seed as argument for random generation stability?

**Comment by AlexKoff88** [comment]
> > Test is failed due to the known issue "packages/openvino_genai/libopenvino_tokenizers.so: cannot open shared object file: No such file or directory"
> 
> Do we have a ticket for it? I have not seen such issue

I am not sure if the issue is filled already but we discussed this recently in the chat. One of the reasons why the issue is observed here is that nightly packages are used in WWB tests.

File: tools/who_what_benchmark/whowhatbench/text2image_evaluator.py (hunk 1 above)
Comment (eaidova) [reply tools/who_what_benchmark/whowhatbench/text2image_evaluator.py:38]: .item() needed here

`
>>> generator_out = torch.normal(torch.tensor(0.0), torch.tensor(1.0))
>>> generator_out
tensor(-0.6411)
>>> generator_out.item()
-0.6410552263259888
`

File: tools/who_what_benchmark/tests/test_cli_image.py [hunk 4]
```diff
@@ -58,6 +60,64 @@ def test_image_model_types(model_id, model_type, backend):
...
+    ("model_id", "model_type"),
+    [
+        ("echarlaix/tiny-random-stable-diffusion-xl", "text-to-image"),
+    ],
+)
+def test_image_model_genai(model_id, model_type):
+    GT_FILE = "test_sd.json"
+    MODEL_PATH = tempfile.TemporaryDirectory().name
+
+    result = subprocess.run(["optimum-cli", "export",
```
[the remark this answers is not in the document]
Comment (eaidova) [reply tools/who_what_benchmark/tests/test_cli_image.py:73]: that is probably due to issue with save_pretrained produced config with wrong class names, that should be fixed here:
https://github.com/huggingface/optimum-intel/pull/961

File: tools/who_what_benchmark/whowhatbench/wwb.py (hunk 3 above)
Comment (AlexKoff88) [reply tools/who_what_benchmark/whowhatbench/wwb.py:306]: thanks, now the seed is fixed and I was planning to expose it in the follow-up PRs but can do it now

File: tools/who_what_benchmark/whowhatbench/text2image_evaluator.py (hunk 1 above)
Comment (AlexKoff88) [reply tools/who_what_benchmark/whowhatbench/text2image_evaluator.py:38]: thanks. I added but it does not have any effect on the generation results.

**Review by eaidova — APPROVED** [review]
